membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -220,10 +222,59 @@ def universe_dummy(self):
                       [0., 200., 150.], [100., 200., 150.], [200., 200., 150.]])
 
         u = mda.Universe(a, n_atoms=9)
-        u.dimensions = [3, 3, 3,  90., 90., 90.]
+        u.dimensions = [300, 300, 300,  90., 90., 90.]
+
+        return u
+
+    @pytest.fixture()
+    def universe_dummy_full(self):
+        a = np.array([[0., 0., 150.], [100., 0., 150.], [200., 0., 150.],
+                      [0., 200., 150.], [100., 200., 120.], [200., 200., 120.],
+                      [0., 100., 120.], [100., 100., 120.], [200., 100., 120.]])
+
+        u = mda.Universe(a, n_atoms=9)
+        u.dimensions = [300, 300, 300,  90., 90., 90.]
 
         return u
 
+    @pytest.fixture()
+    def universe_dummy_wrap(self):
+        #   +-----------+
+        #   |   | 7 | 8 | 9
+        #   +-----------+
+        # 4 | 5 | 6 |   |
+        #   +-----------+
+        #   |   | 1 | 2 | 3
+        #   +-----------+
+        a = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
+                      [0., 100., 150.], [100., 100., 150.], [-100., 100., 150.],
+                      [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])
```

Comment:
I'm not sure this corresponds to the picture (assuming it goes 1 to 9). Shouldn't 1 wrap from `[300, 0, 110]` to `[0, 0, 110]`? If so, I don't understand how either the unwrapped or wrapped version of atom 1 is in the center of the bottom row?

```python
>>> import MDAnalysis as mda
>>> import numpy as np
>>> a = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
...               [0., 100., 150.], [100., 100., 150.], [-100., 100., 150.],
...               [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])
>>> u = mda.Universe(a, n_atoms=9)
>>> u.dimensions = [300, 300, 300, 90, 90, 90]
>>> ag2 = u.atoms.wrap(inplace=False)
>>> ag2
array([[  0.,   0., 110.],
       [100.,   0., 150.],
       [200.,   0., 150.],
       [  0., 100., 150.],
       [100., 100., 150.],
       [200., 100., 150.],
       [  0., 200., 110.],
       [100., 200., 150.],
       [200., 200., 150.]], dtype=float32)
```

Reply:
Yes, the indexes in the picture were not accurate. Updated.